### PR TITLE
Project page: design review remediation & styleguide trimming

### PR DESCRIPTION
**Pull request by lauraGgit**
## Summary Checklist
**Page Header**
- [x] Changed based on #3374 

**Text**
- [x] Section pre-titles are bold
- [x] Section pre-titles are ~ `2xs` 
- [x] Section pre-titles have 6 units of space above (48px) and 0.5 units below (4px)

**Sidebar**
- [x] H5s have 1 unit of space above
- [x] Agency name has line-height 1.6 (i.e. matches line-height of other text in sidenav)
- [x] Links are unindented 
- [x] No grey borders above or below links
- [x] No grey background on link hover
- [x] Links have regular link style,  color: `#046B99` and underlined

**CTA**
- [x] Text has letterspacing `ls(1)`
- [x] Look into font-smoothing
- [x] Button has background color `#00CFFF` and font-color `#1C304A`.  (can follow the [ACCENT COOL COLOR ](https://designsystem.digital.gov/components/button/)example for states)

**Related Blog Posts**
- [x] "Related blog posts" has 5 units of space above?
- [x] "See all posts..." link is vertically aligned “Related Blog Posts” and right aligned with the 3rd column
- [x] The H4 titles are `#00547A` and should be be `#046B99` 
- [x] Read more color is `#00547A` and should be be `#046B99` 
- [x] The caret on read more is smaller and vertically centered with the x height of the text.

### CODE REVIEW

File: _sass/_components/lists.scss [hunk 1]
```diff
@@ -35,6 +35,28 @@
...
+
+    &:hover {
+      background-color: unset;
+      color: color('primary-dark');
+    }
+  }
+
+  h5 {
+    @include u-padding-top(1);
+    @include u-padding-bottom(2);
```
Comment (igorkorenfeld) [inline _sass/_components/lists.scss:56]: Could we switch these two, so it's `u-padding-top(2)` and `u-padding-bottom(1)`

File: _sass/_components/lists.scss [hunk 2]
```diff
@@ -35,6 +35,28 @@
...
-.partner-info h5 {
-    @include u-padding-y(2);
+.partner-info {
+  a {
+    color: color('accent-cool');
+    text-decoration: underline;
+  }
+
+  .usa-sidenav__sublist a {
+    padding-left: 0;
```
Comment (igorkorenfeld) [inline _sass/_components/lists.scss:45]: could we replace this one with `padding: units(.5) o`

**Review by igorkorenfeld — CHANGES_REQUESTED** [review]
Looking very nice, just a couple of notes

## Main Text
- Text content should align with top of the sidebar
- Pre-headers should be ` theme-color-base-dark` \ `gray-cool-60`

image

## Sidebar
Spacing is a little off because we want the H5s to be closer to the their corresponding links and further from the other H5s. I suggested some inline edits to get to that. Here is what I am aiming for: 
image

File: _sass/_components/lists.scss (hunk 1 above)
Comment (lauraGgit) [reply _sass/_components/lists.scss:56]: 👍

File: _sass/_components/lists.scss (hunk 2 above)
Comment (lauraGgit) [reply _sass/_components/lists.scss:45]: 👍

**Comment by lauraGgit** [comment]
@igorkorenfeld easy-peasy. Just pushed up some updates on that.

**Comment by igorkorenfeld** [comment]
Nice the changes look good! I just noticed that the template doesn't seem to working for the other projects. 

For example the [DOJ project](https://federalist-c58f58dc-a215-4616-a54c-12b5eb011096.app.cloud.gov/preview/18f/18f.gsa.gov/project-page-updates-and-blog/what-we-deliver/doj-crt/), instead of being in the sidebar the project details appear on the bottom:
image

**Comment by lauraGgit** [comment]
so weird! i'll take a look.

**Review by igorkorenfeld — APPROVED** [review]
Looks good!

**Pull request merged**